src/OpenTelemetry.Api/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Added `CompositePropagator` that accepts a list of `ITextFormat` following
```

Comment:
This is good. 
We also need an entry to say the ITextFormat API has changed. Thats more crucial as its a breaking change from previous beta.

Reply:
just updated, and included the new method that we will have to implement